### ISSUE TITLE
Wish: Package Manager "Install Without Dependencies" option

### ISSUE THREAD

**Issue by LukeyJohnson** [DynamoDS/Dynamo#7509]
We should be able to say to the Package Manager "install this package but don't bother trying to install or reinstall packages". 

This would simplify things for a lot of users. As the dependency workflow is not bulletproof enough yet, we need this workaround until it is.

## Dynamo version

1.2.1.3803

## Operating system

Windows 10

## What did you do? 

Try to install or upload packages

## What did you expect to see?

Successes

## What did you see instead?
Failures

**Comment by mjkkirschner** [comment]
@QilongTang @reddyashish @scottmitchell  seems relevant to consider this now.

**Comment by QilongTang** [comment]
@mjkkirschner I agree that making it optional to install all dependencies would be nice
